lib/phoenix_live_view/router.ex

```diff
@@ -48,11 +48,15 @@ defmodule Phoenix.LiveView.Router do
   The current action will always be available inside the LiveView as
   the `@live_action` assign. `@live_action` will be `nil`
   if no action is given on the route definition.
+
   ## Options
 
-    * `:session` - a map of strings keys and values to be merged into the session.
```

Comment:
Was unclear to me here if both keys and values needed to be strings, so I checked and then clarified.